Modules/Core/Common/test/itkXMLFileOutputWindowTest.cxx

```diff
@@ -46,6 +56,33 @@ itkXMLFileOutputWindowTest(int argc, char * argv[])
 
   logger->SetInstance(logger);
 
+  // If not input filename is provided, remove the contents in the existing file to avoid counting existing lines when
+  // contents are appended
+  if (argc == 1)
+  {
+    // In order to initialize the filename, some text needs to be written first
```

Comment:
Does not look like an ideal requirement, but that is beyond the scope of this PR. Feel free to propose a PR for the class to be released from this constraint.